util/include/util/hash.h

```diff
@@ -5,7 +5,7 @@
 #include <string>
 #include <sstream>
 
-#include <boost/uuid/sha1.hpp>
+#include <boost/uuid/detail/sha1.hpp>
```

Comment:
CodeCompass does not compile with this under Xenial and Ubuntu, according to the CI.

Reply:
Yes, I created this pull request before I noticed that #393 also contains similar improvements. Maybe we could consider that PR as a precondition of this one. I'll rebase it after that one is merged.

Reply:
Isn't this issue related to the Boost version instead? The path `boost/uuid/detail/sha1.hpp` available since version 1.68 as I have checked. Under Ubuntu 18.04 the version 1.65 is installed from the APT repository.

Reply:
Fixed by rebase over #393